### PR TITLE
rsx-debug: Dump some more opcodes.

### PR DESCRIPTION
**Pull request by vlj**

### CODE REVIEW

**Comment by MoochMcGee** [comment]
lgtm

**Comment by vlj** [comment]
Removed extra comment and fixed vertex output mask dump loop count.

**Comment by vlj** [comment]
Rebased

File: rpcs3/Emu/RSX/GCM.cpp [hunk 1]
```diff
@@ -704,6 +704,22 @@ namespace
 		{ NV4097_SET_VERTEX_ATTRIB_INPUT_MASK, "NV4097_SET_VERTEX_ATTRIB_INPUT_MASK" },
 		{ NV4097_SET_VERTEX_ATTRIB_OUTPUT_MASK, "NV4097_SET_VERTEX_ATTRIB_OUTPUT_MASK" },
 		{ NV4097_SET_TRANSFORM_BRANCH_BITS, "NV4097_SET_TRANSFORM_BRANCH_BITS" },
+
+#define PARSE_REG(reg) { reg, #reg },
+		PARSE_REG(NV3062_SET_OBJECT)
+
+			PARSE_REG(NV3062_SET_CONTEXT_DMA_NOTIFIES)
```
Comment (danilaml) [inline rpcs3/Emu/RSX/GCM.cpp]: Why weird formatting and macros only for these values though?

**Comment by vlj** [comment]
The macro could be used for every other pair too, I used it only to save typing.

**Comment by vlj** [comment]
Rebased

**Comment by vlj** [comment]
Rebased + added some antialiasing related dump + removed the PARSE_REG macro.

**Pull request merged**